.github/workflows/docs-language-linter.yml

```diff
@@ -14,4 +14,5 @@ jobs:
       - uses: actions/checkout@v4
       - uses: errata-ai/vale-action@reviewdog
         with:
+          fail_on_error: false
```

Comment:
i just added this check because vale shouldn't fail our PRs but it should highlight the grammar mistakes